### ISSUE TITLE
(feature suggestion) ability to select an existing chat or create a new chat. 

### ISSUE THREAD

**Issue by ghost** [realcoloride/node_characterai#98]
Straightforward:

I'd like to have the current ``createOrContinueChat()`` method separated into following two methods:
- select an already existing chat
- create an entirely new chat

What led to this suggestion is that I attempted to reset the progress of my character.ai character by trying out the given settings provided by the official character.ai site. 

Despite trying to “save & start a new chat“, the ``createOrContinueChat()`` method seems to prioritize a particular chat that has already been created by the method, and attempting to create a new chat in hope that the new one will be prioritized doesn't seem to work either.

To summarize: 
neither does the site nor node_characterai provide an option to start a completely fresh chat/delete the existing/saved ones (unless I'm blind) 

The only solution I could come up with is to create a new account in order to generate a new key as the new account would then be entirely fresh with no progress whatsoever.

Feel free to correct me on this regard.
